app/routes/endemics/you-can-claim-multiple.js

```diff
@@ -0,0 +1,71 @@
+const session = require('../../session')
+const { agreeSameSpecies, organisation: organisationKey } = require('../../session/keys').farmerApplyData
+const config = require('../../config/index')
+const urlPrefix = require('../../config/index').urlPrefix
+const {
+  endemicsNumbers,
+  endemicsYouCanClaimMultiple,
+  endemicsCheckDetails,
+  endemicsOfferRejected
+} = require('../../config/routes')
+
+const pageUrl = `${urlPrefix}/${endemicsYouCanClaimMultiple}`
+const backLink = `${urlPrefix}/${endemicsCheckDetails}`
+const nextPage = `${urlPrefix}/${endemicsNumbers}`
+
+const agreeStatusValue = 'agree'
+const agreementStatuses = [
+  {
+    value: agreeStatusValue,
+    text: 'I agree',
+    storedAnswer: 'yes'
+  },
+  {
+    value: 'notAgree',
+    text: 'I do not agree',
+    storedAnswer: 'no'
+  }
+]
+
+module.exports = [
+  {
+    method: 'GET',
+    path: pageUrl,
+    options: {
+      handler: async (request, h) => {
+        const organisation = session.getFarmerApplyData(request, organisationKey)
+        return h.view(endemicsYouCanClaimMultiple, {
+          backLink,
+          agreementStatus: agreementStatuses,
+          organisation
+        })
+      }
+    }
+  },
+  {
+    method: 'POST',
+    path: pageUrl,
+    options: {
+      handler: async (request, h) => {
+        const status = agreementStatuses.find((s) => s.value === request.payload.agreementStatus)
+
+        session.setFarmerApplyData(
+          request,
+          agreeSameSpecies, // NOTE AHWR-427 switch to agreeMultipleSpecies, once MI report supports it
+          status.storedAnswer
+        )
+
+        if (status.value === agreeStatusValue) {
+          return h.redirect(nextPage)
+        } else {
```

Comment:
Minor thing, but I dont think we need the else here. Could help with readability if we removed it and added a line to separate when `status.value === agreeStatusValue `and when it doesnt.

Reply:
Agreed, will change shortly

Reply:
yep no need for an else if you are making an early return.